syntax = "proto2";

package janitor;

message CodebaseMetadata {
   optional string name = 1;

   optional string branch_url = 2;
   optional string subpath = 3;
};

message CodebaseList {
    repeated CodebaseMetadata codebase = 1;
};
